### PR TITLE
fix(curriculum): change in broken link

### LINKED ISSUE #44293: Broken link on Scientific Computing with Python project pages
## Affected pages
The link to the Learn Python Video Course is broken on these  pages:

* [Arithmetic Formatter](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter)
* [Time Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/time-calculator)
* [Budget App](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/budget-app)
* [Polygon Area Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/polygon-area-calculator)
* [Probability Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator)

## Your code

```




```

## Expected behavior

When a user clicks on the link "Learn Python Video Course", the home page for the course should be displayed. 

## Screenshots

Screen Shot 2021-11-26 at 8 30 44 AM

## System
- Device: laptop
- OS: macOS Catalina 10.15.7
- Browser: Chrome
- Version: 96.0.4664.55

### PR DESCRIPTION
**Pull request by gourcool**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44293

### CODE REVIEW

**Review by gikf — CHANGES_REQUESTED** [review]
Could you make similar change to the other python projects in _Scientific Computing with Python Projects_?

Note for future - it is recommended to make changes on a separate branch, than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally.

File: curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ You will be [working on this project with our Replit starter code](https://repli
 We are still developing the interactive instructional part of the Python curriculum. For now, here are some videos on the freeCodeCamp.org YouTube channel that will teach you everything you need to know to complete this project:
 
 - [Python for Everybody Video Course](https://www.freecodecamp.org/news/python-for-everybody/) (14 hours)
-- [Learn Python Video Course](https://www.freecodecamp.org/news/learn-python-video-course/) (10 hours)
+- [Learn Python Video Course](https://www.freecodecamp.org/news/learn-python-basics-in-depth-video-course/) (10 hours)
```
[the remark this answers is not in the document]
Comment (gourcool) [reply curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter.md:16]: Can you please assign these changes to me I happy to help

[the remark this answers is not in the document]
Comment (gikf) [reply curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter.md]: I'm not able to suggest changes to those files, like to one above, as they weren't changed in the PR yet, and github doesn't give such option.

You can add changes in new commit and push it to the same branch, the PR will be updated with new commit.

[the remark this answers is not in the document]
Comment (gourcool) [reply curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter.md]: ok thanks

**Comment by Mateusza-jpg** [comment]
Część potrzebuje pomocy


wt., 30 lis 2021, 10:31 użytkownik Krzysztof G. ***@***.***>
napisał:

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you, for taking the time to open this.

However, it seems we might be going with a different solution for all of the challenges. I will leave this open for now, until we have decided on the best way to resolve the issue.

**Comment by nhcarrigan** [comment]
@ShaunSHamilton do you recall where that discussion was? And what the plan for this PR is?

**Comment by ShaunSHamilton** [comment]
@nhcarrigan The discussion was on the linked issue: https://github.com/freeCodeCamp/freeCodeCamp/issues/44293#issuecomment-988845198

**Comment by nhcarrigan** [comment]
Thanks @ShaunSHamilton you're the best 🙂 

@gourcool apologies for the delay on reviewing this contribution. It looks like the consensus is to remove the "We are still developing..." text, as well as the subsequent links. Would you be interested in updating your PR to reflect that?

**Comment by ahmadabdolsaheb** [comment]
@gourcool, It would be great if we could update the pr based on the above feedback so we could move forward with merging your solution.

Let us know if you need any help.

**Pull request closed without merging**